src/SearchResults.js

```diff
@@ -0,0 +1,86 @@
+import CustomerProfile from "./CustomerProfile";
+import React, { useState } from "react";
+import moment from "moment";
+
+const TableRow = ({ booking, handleShowProfile }) => {
+  const [isSelected, setIsSelected] = useState(false);
+
+  const toggleHighlight = () => {
+    setIsSelected(!isSelected);
+  };
+
+  const highlightClass = isSelected ? "highlight" : "";
+
+  return (
+    <tr onClick={toggleHighlight} className={highlightClass}>
+      <th scope="row">{booking.id}</th>
+      <td>{booking.title}</td>
+      <td>{booking.firstName}</td>
+      <td>{booking.surname}</td>
+      <td>{booking.email}</td>
+      <td>{booking.roomId}</td>
+      <td>{booking.checkInDate}</td>
+      <td>{booking.checkOutDate}</td>
```

Comment:
you can destructure the booking object so you dont have to use `booking.` on every line